oschecks/check/check_gnocchi.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+from gnocchiclient import client as gnocchi_client
+from gnocchiclient import exceptions as gnocchi_exceptions
+
+import oschecks.openstack as openstack
+import oschecks.common as common
+
+
+class GnocchiCommand(openstack.OpenstackCommand):
+    '''Base class for Gnocchi checks.'''
+
+    def get_parser(self, prog_name):
+        parser = super(GnocchiCommand, self).get_parser(prog_name)
+        if not hasattr(self, 'arg_group'):
```

Comment:
```
   >>> import argparse 
   >>> parser = argparse.ArgumentParser(description='Das Test')
   >>> parser.add_argument_group('Einz').add_argument('-Einz')
   _StoreAction(option_strings=['-Einz'], dest='Einz', nargs=None, const=None, default=None, type=None, choices=None, help=None, metavar=None)
   >>> parser.add_argument_group('Einz').add_argument('-Zwei')
  _StoreAction(option_strings=['-Zwei'], dest='Zwei', nargs=None, const=None, default=None, type=None, choices=None, help=None, metavar=None)
   >>> parser.print_help()
   usage: [-h] [-Einz EINZ] [-Zwei ZWEI]

   Das Test

   optional arguments:
     -h, --help  show this help message and exit

   Einz:
     -Einz EINZ

   Einz:
     -Zwei ZWEI
```
  



You see, the same group would be created twice (First in calling GnocchiCommand.get_parser and second in calling CheckResourceTypeExists.get_parser(). Hence we need to make sure, that if the base class has the group created already, child classes should not create their own.